### DISCUSSION TITLE
Cannot find module error (only on Vercel) with Drizzle ORM

### DISCUSSION THREAD

**Discussion by zach-hopkins** [vercel/community#3007] [Help]
### Summary

I am getting the below very frustrating issue when I deploy to vercel (serverless /api routes). It works just fine when ran locally. I've deployed on a separate project to see if it was a cache thing, nada. 

`2023-06-26T16:26:31.061Z	undefined	ERROR	Cannot find module '/var/task/node_modules/drizzle-orm/postgres-js/index.cjs'
2023-06-26T16:26:31.075Z	undefined	ERROR	Did you forget to add it to "dependencies" in `package.json`?
RequestId: 0fb1b5a1-141e-4a38-891a-bbe71276563b Error: Runtime exited with error: exit status 1
Runtime.ExitError`

Here is an example endpoint:

```
import { drizzle } from "drizzle-orm/postgres-js";
import postgres from "postgres";
import * as schema from "../drizzle/schema";

const connectionString = process.env.DATABASE_URL;
const client = postgres(connectionString);

const db = drizzle(client, { schema });

const getCustomers = async (req, res) => {
  try {
    const customers = await db.query.customer.findMany({
      columns: {
        customerid: true,
        customername: true,
        address: true,
        payableemail: true,
        payablephone: true,
        status: true,
        notes: true,
      },
      with: {
        project: {
          columns: {
            projectid: true,
            projectname: true,
          },
        },
      },
    });
    res.send(customers);
  } catch (error) {
    console.log(error);
    res.status(500).send("Internal Server Error");
  }
};

export default getCustomers;
```

and here is package.json

`{
  "name": "dashboard",
  "version": "0.0.0",
  "scripts": {
    "frontend": "vite",
    "build": "vite build",
    "preview": "vite preview --port 4173",
    "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs --fix --ignore-path .gitignore"
  },
  "dependencies": {
    "@fontsource/inter": "^4.5.12",
    "@fontsource/poppins": "^4.5.9",
    "@headlessui/vue": "^1.6.7",
    "@heroicons/vue": "^2.0.10",
    "@prisma/client": "^4.16.0",
    "@supabase/supabase-js": "^2.26.0",
    "@tailwindcss/forms": "^0.5.2",
    "axios": "^1.4.0",
    "bcryptjs": "^2.4.3",
    "drizzle-orm": "^0.27.0",
    "eslint-plugin-prettier": "^4.2.1",
    "jsonwebtoken": "^8.5.1",
    "moment": "^2.29.4",
    "postgres": "^3.3.5",
    "qs": "^6.11.2",
    "tailwindcss": "^3.1.8",
    "v-tooltip": "^2.1.3",
    "vue": "^3.2.38",
    "vue-eslint-parser": "^9.0.3",
    "vue-final-modal": "^3.4.4",
    "vue-router": "^4.1.5",
    "vue-toastification": "2.0.0-rc.5"
  },
  "devDependencies": {
    "@rushstack/eslint-patch": "^1.1.4",
    "@types/qs": "^6.9.7",
    "@vitejs/plugin-vue": "^3.0.3",
    "@vue/eslint-config-prettier": "^7.0.0",
    "autoprefixer": "^10.4.8",
    "drizzle-kit": "^0.19.2",
    "eslint": "^8.23.0",
    "eslint-plugin-vue": "^9.4.0",
    "postcss": "^8.4.16",
    "prettier": "^2.7.1",
    "prisma": "^4.16.0",
    "vite": "^3.0.9"
  }
}`

### Steps to Reproduce

Test with vercel dev > publish to github > go to api endpoint

**Comment by amyegan** [comment]
I just tried deploying a new project from the [Vercel Postgres + Drizzle Next.js Starter](https://vercel.com/templates/next.js/postgres-drizzle) without getting any errors. It looks like this repo does things a little differently though.

If you're able to share your public repo or a minimal reproduction, I can take a closer look.
